code/core/src/phone/java/com/adobe/marketing/mobile/services/ui/message/views/MessageFrame.kt

```diff
@@ -115,6 +117,21 @@ internal fun MessageFrame(
         Row(
             modifier = Modifier
                 .fillMaxSize()
+                .onPlaced {
+                    heightDp.value = with(density) { ((contentView.height.toDp() * inAppMessageSettings.height) / 100) }
+                    widthDp.value = with(density) { ((contentView.width.toDp() * inAppMessageSettings.width) / 100) }
+                    Log.debug(
+                        ServiceConstants.LOG_TAG,
+                        "MessageFrame",
+                        "Placed message frame with height: ${heightDp.value} and width: ${widthDp.value}"
+                    )
+
+                    Log.debug(
+                        ServiceConstants.LOG_TAG,
+                        "MessageFrame",
+                        "Placed message frame with content height: ${contentView.height} and width: ${contentView.width}"
+                    )
```

Comment:
This will add a lot of noise to the logs due to recompositions. Let us  not log from these Composables